src/components/PreLoader/PreLoader.ts

```diff
@@ -20,15 +20,15 @@ export class PreLoader implements PreLoaderInterface {
     public async hide() {
         await sleep(HIDE_DURATION);
 
-        this.loader && this.loader.setAttribute("class", "loaded");
+        this.loader && document.body.setAttribute("class", "loaded");
```

Comment:
Will have conflicts with another scripts that modify body classes.
Consider `document.body.classList` usage